lib/cfg.c

```diff
@@ -549,7 +549,8 @@ _load_file_into_string(const gchar *fname)
                 evt_tag_str(EVT_TAG_OSERROR, error->message));
 
       g_error_free(error);
-      return content;
+      g_string_free(content, TRUE);
```

Comment:
You can remove this line and the allocation itself too if you move `g_string_new()` a few lines below.

What do you mean by "root"?

Reply:
sure did in the last patch.
root: Config not included, nor generated. (user provided directly to use)